proto/lib/config/optionsconfiguration.proto

```diff
@@ -30,7 +30,6 @@ message OptionsConfiguration {
     string          usage_reporting_url                      = 21 [(ext.goname) = "URURL", (ext.xml) = "urURL", (ext.json) = "urURL", (ext.default) = "https://data.syncthing.net/newdata"];
     bool            usage_reporting_post_insecurely          = 22 [(ext.goname) = "URPostInsecurely", (ext.xml) = "urPostInsecurely", (ext.json) = "urPostInsecurely", (ext.default) = "false"];
     int32           usage_reporting_initial_delay_s          = 23 [(ext.goname) = "URInitialDelayS", (ext.xml) = "urInitialDelayS", (ext.json) = "urInitialDelayS", (ext.default) = "1800"];
-    bool            restart_on_wakeup                        = 24 [(ext.default) = "true"];
```

Comment:
Should not be removed but moved down to the deprecated options for backwards compost
 Likely doesn't matter as json/xml ignoring unknown entries, but would still be good to handle it given we use proto.

Reply:
I considered a `reserved 24;` but skipped it since we have no binary marshalling of these types... Removing it entirely means it will just get dropped on next save, which seems fine to me?

Reply:
yah right, it really doesn't matter and what i intended to propose was actually wrong. all good as is here